### DISCUSSION TITLE
cstring과 string의 혼동

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#72] [Q&A] [answered]
## code
https://github.com/Woon-2/cpp-study/blob/4799a142ad15e906151932e442905d50c0a7db58/assignment-sprint1-write-cpp-classes/assignment/Project1/my_string.h#L1-L34

## 헷갈린 점
std::string에 컴파일 오류가 떠서 봤더니 cstring은 문자열을 다루는 함수가 들어있지 basic_string이 아니였습니다..

따라서 헤더파일을 수정할 예정입니다.

**Comment by Woon-2** [comment, marked as answer]
좋습니다.   
`my_string`은 `std::string`을 모방하려고 만든 클래스이니 `std::string`을 이용하지 않고 만들어져야 합니다.
